packages/atomic/package.json

```diff
@@ -20,12 +20,15 @@
     "dist/",
     "loader/",
     "docs/",
-    "licenses/"
+    "licenses/",
+    "headless/"
   ],
   "scripts": {
+    "clean": "rimraf -f -r dist/* headless/*",
     "build": "npm run build:locales && npm run build:stencil && npm run doc:list-assets && npm run build:storybook",
     "build:locales": "node --max-old-space-size=4096 ./scripts/split-locales.js",
-    "build:stencil": "node --max_old_space_size=6144 ./node_modules/@stencil/core/bin/stencil build",
+    "build:stencil": "npm run copy:headless && node --max_old_space_size=6144 ./node_modules/@stencil/core/bin/stencil build",
+    "copy:headless": "ncp ../headless/ ./headless/ && node scripts/set-headless-private.js",
```

Comment:
I believe copying from `../headless` will include a bunch of directories and files that are not needed, but that will show up when importing sub-packages from `@coveo/atomic/headless/...`.

Is using `@coveo/headless` from `node_modules` not possible?

Reply:
@coveo/headless from node_modules is packages/headless symlinked

Reply:
I see, so it's a problem with both approaches ...

When running [npm publish](https://docs.npmjs.com/cli/v8/commands/npm-publish#files-included-in-package), only the paths in the `files` array and a few other files like the `package.json` are published to npm. It's possible to see the list by running `npm publish --dry-run`.

Ideally, we would upload just what's needed, and keep a consistent structure as what we are pushing to npm.

Reply:
I've updated the files property in the package.json to cover this. I thought it would have been managed automatically since the headless/package.json also as a file property...